watertap/property_models/ion_DSPMDE_prop_pack.py

```diff
@@ -160,13 +187,58 @@ def build(self):
             initialize=80.4, #todo: make a variable with parameter values for coefficients in the function of temperature
             units=pyunits.dimensionless,
             doc="Dielectric constant of water")
-
+        self.debye_huckel_b = Param(
+            mutable=True,
+            default=0.3,
+            initialize=0.3,
+            units=pyunits.kg/pyunits.mol,
+            doc="Debye Huckel constant b")
+
+        # Mass density parameters, eq. 8 in Sharqawy et al. (2010)
+        dens_units = pyunits.kg / pyunits.m ** 3
+        t_inv_units = pyunits.K ** -1
+
+        self.dens_mass_param_A1 = Var(
+            within=Reals, initialize=9.999e2, units=dens_units,
+            doc='Mass density parameter A1')
+        self.dens_mass_param_A2 = Var(
+            within=Reals, initialize=2.034e-2, units=dens_units * t_inv_units,
+            doc='Mass density parameter A2')
+        self.dens_mass_param_A3 = Var(
+            within=Reals, initialize=-6.162e-3, units=dens_units * t_inv_units ** 2,
+            doc='Mass density parameter A3')
+        self.dens_mass_param_A4 = Var(
+            within=Reals, initialize=2.261e-5, units=dens_units * t_inv_units ** 3,
+            doc='Mass density parameter A4')
+        self.dens_mass_param_A5 = Var(
+            within=Reals, initialize=-4.657e-8, units=dens_units * t_inv_units ** 4,
+            doc='Mass density parameter A5')
+        self.dens_mass_param_B1 = Var(
+            within=Reals, initialize=8.020e2, units=dens_units,
+            doc='Mass density parameter B1')
+        self.dens_mass_param_B2 = Var(
+            within=Reals, initialize=-2.001, units=dens_units * t_inv_units,
+            doc='Mass density parameter B2')
+        self.dens_mass_param_B3 = Var(
+            within=Reals, initialize=1.677e-2, units=dens_units * t_inv_units ** 2,
+            doc='Mass density parameter B3')
+        self.dens_mass_param_B4 = Var(
+            within=Reals, initialize=-3.060e-5, units=dens_units * t_inv_units ** 3,
+            doc='Mass density parameter B4')
+        self.dens_mass_param_B5 = Var(
+            within=Reals, initialize=-1.613e-5, units=dens_units * t_inv_units ** 2,
+            doc='Mass density parameter B5')
+
+        for v in self.component_objects(Var):
+            v.fix()
```

Comment:
Note that this will *also* fix any `Var`s created by the call to `super().build()` above, or, alternatively, all the `Var`s currently on the block.

Probably doesn't matter in this case, but we might consider being more explicit here.

Reply:
This is the way we usually do it (e.g., in NaCl and seawater properties) although there is a comment that goes with this: 
`# traditional parameters are the only Vars currently on the block and should be fixed`